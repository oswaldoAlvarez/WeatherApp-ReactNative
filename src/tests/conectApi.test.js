import React from 'react';
import renderer from 'react-test-renderer';

test('Probando funciones del conectApi', () => {
  const tree = renderer.create(<conectApi />).toJSON();
  expect(tree).toMatchSnapshot();
});
