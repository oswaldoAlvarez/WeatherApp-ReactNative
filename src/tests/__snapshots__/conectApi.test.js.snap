// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando funciones del conectApi 1`] = `<conectApi />`;
